system/include/webgpu/webgpu.h

```diff
@@ -269,9 +291,19 @@ typedef enum WGPUSType {
     WGPUSType_SurfaceDescriptorFromCanvasHTMLSelector = 0x00000004,
     WGPUSType_ShaderModuleSPIRVDescriptor = 0x00000005,
     WGPUSType_ShaderModuleWGSLDescriptor = 0x00000006,
+    WGPUSType_SamplerDescriptorDummyAnisotropicFiltering = 0x00000007,
+    WGPUSType_RenderPipelineDescriptorDummyExtension = 0x00000008,
```

Comment:
These are Dawn testing cruft. (Dawn should eventually move them.)
```suggestion
```

Reply:
Removed in all files